src/cmd/build.js

```diff
@@ -53,10 +54,10 @@ export async function getDefaultLocalizedName(
   let extensionName: string = manifestData.name;
 
   try {
-    messageData = JSON.parse(await fs.readFile(messageFile));
+    messageData = parseJson(await fs.readFile(messageFile));
   } catch (error) {
-    throw new UsageError(
-      `Error reading or parsing file ${messageFile}: ${error}`);
+    error.fileName = messageFile;
+    throw new UsageError(error);
```

Comment:
`UsageError` constructor is supposed to receive a `string` error message as its only parameter, while the above line set the error message to an error object, and so it should be changed into something like: 
```js
  throw new UsageError(`Error parsing locale file at ${messageFile}: ${error}`);
```